scripts/_common.sh

```
@@ -0,0 +1,680 @@
+  fi
+}
+
+# Shows an error if required tools are not installed.
+function required {
+  local e=0
+  for tool in "$@"; do
+    type $tool >/dev/null 2>&1 || {
+      e=1 && error "$tool is required"
+    }
+  done
+  [[ $e < 1 ]] || fatal "please install missing tools required for running this script and try again"
```

Shellcheck (0.10.0)

[error] < is for string comparisons. Use -lt instead.

(SC2071)